docs/UserGuide.md

```diff
@@ -219,10 +219,10 @@ Examples:
 
 Adds an order to ReadyBakey’s order list. 
 
-Format: `addo p/PHONE r/REMARK d/DETAILS c/DELIVERYDATETIME m/COLLECTION_TYPE`
+Format: `addo p/PHONE d/DETAILS c/DELIVERYDATETIME m/COLLECTION_TYPE [r/REMARK]…`
```

Comment:
if you are putting the square brackets I think we should keep it consistent throughout the UG? Also, details was missed. Should have the `...`?

Reply:
I think this will require further discussion on the appropriate way to handle this situation, as ... would mean that DETAILS can be 0, which it cannot actually be. We might need a new symbol to handle this.